queries/javascript/folds.scm

```diff
@@ -1,20 +1 @@
-; inherits: (jsx)
-[
-  (for_in_statement)
-  (for_statement)
-  (while_statement)
-  (arrow_function)
-  (function)
-  (function_declaration)
-  (class_declaration)
-  (method_definition)
-  (do_statement)
-  (with_statement)
-  (switch_statement)
-  (switch_case)
-  (import_statement)
-  (if_statement)
-  (try_statement)
-  (catch_clause)
-  (object)
-] @fold
+; inherits: ecma,jsx
```

Comment:
Won’t this include all rules for JSX?  
JS code is not necessarily JSX.

Reply:
That's true but I believe you can write JSX code in a file with a .jsx extension. Since they use the same parser I think this will be ok.

Reply:
yep you can write jsx in js